### DISCUSSION TITLE
EmojiMart for Reactions and cross-platform compatibility

### DISCUSSION THREAD

**Discussion by synth** [missive/emoji-mart#786] [Q&A]
We want to implement EmojiMart as a picker to add Slack-style reactions in our web application. Reactions will also be available in our mobile app and API. Once an emoji is picked as a reaction, we then save the reaction to our database and have to choose how to save it. Saving the native emoji Unicode is not desirable as detailed here: https://nolanlawson.com/2022/04/08/the-struggle-of-using-native-emoji-on-the-web/. 

So, we then want to save an identifier to that emoji, presumably a short code. We could use EmojiMart's ids, but then this becomes difficult on our mobile app and API because we'd have to somehow map EmojiMart's ids to the mobile app's pickers ids. Additionally, we don't know the ids that may be best to use for API clients.

We'd much prefer to use ids that are commonplace, like the ones published on WebFX and used by many major platforms here:  https://www.webfx.com/tools/emoji-cheat-sheet/

Is anyone else using EmojiMart for "reactions" and doing so in a cross-platform way? Is there a way to utilize WebFx's shortcodes with EmojiMart?

**Comment by Simon-Laux** [comment]
you could probably provide your own data json file

**Reply by synth** [reply]
We considered this, but it is not without issues. The preference would be to have something officially supported. We would be happy to create a new data json file and submit a PR, but wanted to know if this would have a shot at being upstreamed.
